mavros/src/lib/mavros.cpp

```diff
@@ -96,23 +96,24 @@ MavRos::MavRos() :
 		fcu_link->set_protocol_version(mavconn::Protocol::V10);
 	}
 
-	if (gcs_url != "") {
-		ROS_INFO_STREAM("GCS URL: " << gcs_url);
-		try {
-			gcs_link = MAVConnInterface::open_url(gcs_url, system_id, component_id);
-
-			gcs_link_diag.set_mavconn(gcs_link);
-			gcs_diag_updater.setHardwareID(gcs_url);
-			gcs_diag_updater.add(gcs_link_diag);
-		}
-		catch (mavconn::DeviceError &ex) {
-			ROS_FATAL("GCS: %s", ex.what());
-			ros::shutdown();
-			return;
-		}
-	}
-	else
-		ROS_INFO("GCS bridge disabled");
+    if (gcs_url != "") {
```

Comment:
Indentation are 8 spaces, not 4. Please use uncrustify (`uncrustify -c ${ROS_WORKSPACE}/src/mavros/mavros/tools/uncrustify-cpp.cfg --replace --no-backup <path/to/file.ext>`).